### PR TITLE
updated documentation to add draft PR's #244

### PR DESCRIPTION
**Pull request by sushmaunnibhavi**
Closes # .

Changes proposed in this pull request:

-removed the WIP tag
-added the tag to add draft pull requests

### CODE REVIEW

**Comment by tsalo** [comment]
Thank you for this PR. I have a couple of notes:

1. The associated issue (#221) actually requests the opposite documentation change (i.e., that we ask contributors to use the WIP tag and _not_ to use the draft PR utility). I originally wanted to take advantage of the new draft PR function, but Travis CI does not currently run unit tests on drafts, so we have to explicitly request that contributors avoid it for now.
2. We have a preferred format for PR titles. Would you mind updating yours to match it? We like to have a tag in square brackets before the title. In this case, [DOC] would be most appropriate, since this PR proposes an update to the documentation. After the tag, we have the title in sentence case (e.g., “Update documentation for draft PRs”). There’s also no reason to include the PR number in the title, since GitHub retains that information.
3. Since this pull request is meant to close an existing issue (#221), you should include the line “Closes #221" in the pull request description. The pull request template provides “Closes # .” by default to make it easier.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -136,17 +136,15 @@ When opening a pull request, please use at least one of the following prefixes:
...
-This tells the development team that your pull request is a "work-in-progress",
-and that you plan to continue working on it.
+In order to use draft PR's include the **[DPR]** prefix.
+For more information on draft pull requests visit (https://github.blog/2019-02-14-introducing-draft-pull-requests/)
 
 You can also combine the tags above, for example if you are updating both a test and
 the documentation: **[TST, DOC]**.
-If you're still working on the pull request that prefix would be **[WIP, TST, DOC]**.
-
+If you're still working on the pull request that prefix would be **[TST, DOC]**.
```
Comment (jbteves) [inline CONTRIBUTING.md]: I don't believe we want to make this change per [this comment](https://github.com/ME-ICA/tedana/pull/244#issuecomment-478255157)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -136,17 +136,15 @@ When opening a pull request, please use at least one of the following prefixes:
...
 * **[STY]** for stylistic changes
 * **[TST]** for new or updated tests, and
-* **[WIP]** for changes which are not yet ready to be merged
+* **[DPR]** for creating draft pull requests
 
 Pull requests should be submitted early and often!
-If your pull request is not yet ready to be merged, please also include the **[WIP]** prefix.
-This tells the development team that your pull request is a "work-in-progress",
-and that you plan to continue working on it.
+In order to use draft PR's include the **[DPR]** prefix.
```
Comment (jbteves) [inline CONTRIBUTING.md]: Please adjust per [this comment](https://github.com/ME-ICA/tedana/pull/244#issuecomment-478255157)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -136,17 +136,15 @@ When opening a pull request, please use at least one of the following prefixes:
 * **[REF]** for refactoring existing code
 * **[STY]** for stylistic changes
 * **[TST]** for new or updated tests, and
-* **[WIP]** for changes which are not yet ready to be merged
+* **[DPR]** for creating draft pull requests
```
Comment (jbteves) [inline CONTRIBUTING.md]: Per [this comment](https://github.com/ME-ICA/tedana/pull/244#issuecomment-478255157), please undo this change.
